pkg/client/build.go

```diff
@@ -1206,19 +1256,10 @@ func (c *Client) fetchBuildpack(ctx context.Context, bp string, relativeBaseDir
 			Version: version,
 		}
 	default:
-		builderOS, err := builderImage.OS()
-		if err != nil {
-			return nil, nil, errors.Wrapf(err, "getting builder OS")
-		}
-
-		builderArch, err := builderImage.Architecture()
-		if err != nil {
-			return nil, nil, errors.Wrapf(err, "getting builder architecture")
-		}
-
 		downloadOptions := buildpack.DownloadOptions{
 			RegistryName:    registry,
-			Target:          &dist.Target{OS: builderOS, Arch: builderArch},
+			ImageOS:         targetToUse.OS,
```

Comment:
This seemed to be needed by unit tests

Reply:
I ran locally, and removing that line it's ok for me

Reply:
Removed in 08995083